astropy/coordinates/angle_utilities.py

```diff
@@ -24,6 +29,20 @@
 from .. import units as u
 
 
+TAB_HEADER = """# -*- coding: utf-8 -*-
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+# This file was automatically generated from ply. To re-generate this file,
+# remove it from this folder, then build astropy and run the tests in-place:
+#
+#   python setup.py build_ext --inplace
+#   pytest astropy/coordinates
```

Comment:
how about replacing this test part with: 
```
python -c "from astropy.coordinates import angle_utilities;angle_utilities._AngleParser()"
```
?  That seems a bit more fine-grained

Reply:
Mine is more general in case we add multiple parsers in future :)

Reply:
Hmm, I think it's very dangerous to use the tests to do this... but I see your point that *something* more general is needed.  So I guess this is fine for consistency, but I'm going to make an issue to try to address this better anyway.